testing/ncids-css-testing/stories/uswds-native/usa-collection/src/usa-collection.scenarios.js

```diff
@@ -0,0 +1,191 @@
+module.exports = [
+	{
+		label: 'usa-collection passthough default',
+		storyId: 'uswds-components-collection--default',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-collection passthough fancy',
+		storyId: 'uswds-components-collection--fancy',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-collection passthough media',
+		storyId: 'uswds-components-collection--media',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-collection passthough only headers',
+		storyId: 'uswds-components-collection--only-headers',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-collection passthough default link focus',
+		storyId: 'uswds-components-collection--default',
+		focusSelector: '.usa-link',
```

Comment:
if there are no styles for usa-link in usa-collections, we do not need this many tests for usa-link. usa-link shoud be tested in usa-link.